source/common/html/utility.cc

```diff
@@ -0,0 +1,37 @@
+#include "common/html/utility.h"
+
+#include <string>
+
+namespace Envoy {
+namespace Html {
+
+/* static */
+std::string Utility::sanitize(const std::string& text) {
+  std::string buffer;
+  for (char ch : text) {
```

Comment:
I was curious if absl provided this, since it's one of our external deps. It doesn't, but it does provide a slightly more concise way to write this: https://github.com/abseil/abseil-cpp/blob/cf6ab6bb2b800fae859ccc735f398d22a7336a00/absl/strings/str_replace.h#L32

Reply:
done